core/domain/story_domain.py

```diff
@@ -468,24 +525,38 @@ def validate(self):
                     'The story node with ID %s points to itself.' % node_id)
 
 
+class StoryContentsDict(TypedDict):
+    """Dictionary representing the StoryContents object."""
+
+    nodes: List[StoryNodeDict]
+    initial_node_id: Optional[str]
+    next_node_id: str
+
+
 class StoryContents:
     """Domain object representing the story_contents dict."""
 
-    def __init__(self, story_nodes, initial_node_id, next_node_id):
+    def __init__(
+        self,
+        story_nodes: List[StoryNode],
+        initial_node_id: Optional[str],
+        next_node_id: str
+    ) -> None:
         """Constructs a StoryContents domain object.
 
         Args:
             story_nodes: list(StoryNode). The list of story nodes that are part
                 of this story.
-            initial_node_id: str. The id of the starting node of the story.
+            initial_node_id: Optional[str]. The id of the starting node of the
+                story and None if there is only one node(or the starting node).
```

Comment:
There are some cases in the file where we are defining the initial node and assigning `initial_node_id` as None. 

some of them are:
https://github.com/oppia/oppia/blob/2af5fb246c16ad2f0a944c6577f44898e9109527/core/domain/story_domain.py#L1025
https://github.com/oppia/oppia/blob/2af5fb246c16ad2f0a944c6577f44898e9109527/core/domain/story_domain.py#L1271